panic: runtime error: slice bounds out of range

Following the README I installed via curl but get the following message when I run `doenter`:

```
panic: runtime error: slice bounds out of range

goroutine 6 [running]:
panic(0xeff20, 0xc82000a0b0)
    /usr/local/go/src/runtime/panic.go:481 +0x3e6
main.read(0x2491e8, 0xc820012240)
    /go/src/github.com/fntlnz/doenter/main.go:18 +0x232
created by main.main
    /go/src/github.com/fntlnz/doenter/main.go:54 +0x19b
```

--- a/main.go
+++ b/main.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	"bufio"
-	"fmt"
+	"io"
 	"log"
 	"os"
 	"os/signal"
@@ -14,9 +14,7 @@ import (
 
 func read(s serial.Port) {
 	for {
-		buf := make([]byte, 128)
-		n, _ := s.Read(buf)
-		fmt.Fprintf(os.Stdout, "%s", buf[:n])
+		io.Copy(os.Stdout, s)
 	}
 }
 

